WordPress/Classes/ViewRelated/Reader/ReaderMenuViewController.swift

```diff
@@ -84,9 +84,10 @@ import WordPressShared
             restorableSelectedIndexPath = defaultIndexPath
         }
 
-        cleanupStaleContent(removeAllTopics: false)
+        unflagInUseContent()
```

Comment:
I'm a bit concerned that this is on the application launch execution path. I profiled on an iPad Mini 1, and it "only" added 30 ms, but I imagine it could be more if it had more posts in Core Data. I can't think of a better way to do this though. If it's a problem, core data batch updates could help a bit. I'm not sure how much of a problem this actually is though.